components/Cards/index.js

```diff
@@ -18,3 +18,74 @@
 // </div>
 //
 // Use your function to create a card for each of the articles and add the card to the DOM.
+let articlesArray = [];
+let cardsContainer = document.querySelector('.cards-container');
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+    .then((res) => {
+        console.log ("Res: ", res) 
+        articlesArray = res.data.articles;    
+        let javascriptArt = articlesArray.javascript;
```

Comment:
You can loop through each of the topics (javascript, bootstrap, etc) using Object.entries

Here's a link explaining more. [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

This isn't something you'd be expected to know but it's a good coding style adjustment.

You can also read about how to use a for loop to accomplish the same thing [https://stackoverflow.com/questions/684672/how-do-i-loop-through-or-enumerate-a-javascript-object#:~:text=You%20can%20use%20the%20for,t%20come%20from%20the%20prototype.&text=Using%20the%20new%20Object](https://stackoverflow.com/questions/684672/how-do-i-loop-through-or-enumerate-a-javascript-object#:~:text=You%20can%20use%20the%20for,t%20come%20from%20the%20prototype.&text=Using%20the%20new%20Object).